experimental/packages/otlp-proto-exporter-base/src/platform/browser/OTLPProtoExporterBrowserBase.ts

```diff
@@ -0,0 +1,90 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { diag } from '@opentelemetry/api';
+import { ServiceClientType } from '../types';
+import {
+  OTLPExporterBrowserBase as OTLPExporterBaseMain,
+  OTLPExporterError,
+  OTLPExporterConfigBase,
+  sendWithXhr,
+} from '@opentelemetry/otlp-exporter-base';
+import * as root from '../../generated/root';
+
+interface ExportRequestType<T, R = T & { toJSON: () => unknown }> {
+  create(properties?: T): R;
+  encode(message: T, writer?: protobuf.Writer): protobuf.Writer;
+  decode(reader: protobuf.Reader | Uint8Array, length?: number): R;
+}
+
+/**
+ * Collector Exporter abstract base class
+ */
+export abstract class OTLPProtoExporterBrowserBase<
```

Comment:
Usually the node and browser exports would have the same name and be chosen at compile time. Is there a reason you've chosen to give it a separate name here?/

Reply:
Yeah, we have a section about this rule: https://github.com/open-telemetry/opentelemetry-js/blob/main/CONTRIBUTING.md#platform-conditional-exports.

If it is needed to be exported under different names, it can be placed outside `platform/` to avoid unexpected missing exports.

Reply:
Thanks for pointing it out. I will make the changes.

Reply:
@dyladan @legendecas this doesn't seem to be straightforward. The classes  are named differently in `platform/` in the package `otlp-exporter-base` and they follow different interface - that is, one has `_headers` private member and another has `headers` public member. I am running into errors using the same name in the `platform/` of package `otlp-proto-exporter-base`. The errors manifest in the package `exporter-trace-otlp-proto`.  It looks like the rule of using the same name for classes in `platform/` is not followed already, so I suggest we take this up in a separate PR as it requires changes in multiple packages and requires more care.  Let me know what you think.

Reply:
@dyladan @legendecas can you please comment on the above observation?

Reply:
Sorry for the delayed response. I thought I've hit the comment button. 

Yeah, it is true that the rule has already been violated in the package `otlp-exporter-base`. I think it is fine to defer the renaming since it would not be an easy work without the refactoring in the `otlp-exporter-base` first.